frontend/src/metabase/visualizations/components/Visualization/Visualization.jsx

```diff
@@ -318,7 +318,14 @@ class Visualization extends PureComponent {
   };
 
   onRender = ({ yAxisSplit, warnings = [] } = {}) => {
-    this.setState({ yAxisSplit, warnings });
+    const currentYAxisSplit = this.state.yAxisSplit;
```

Comment:
We seem to already remove `yAxisSplit` in master, but I'll leave it here just to be safe since it may or may not be used.

https://github.com/metabase/metabase/blob/e62bf9b493e335e2c1009ee8f60c873d3f91986c/frontend/src/metabase/visualizations/components/Visualization/Visualization.jsx#L301-L304

This stops the `warnings` from being updated, since the `warnings`s are different, but have the same value under the hood.